scripts/basic_demo.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import sys
+import rospy 
+from cobot_pump_ros.srv import *
+import time
+
+
+def basic_demo():
+    rospy.wait_for_service('startPump')
+    start_pump = rospy.ServiceProxy('startPump', startPump)
+    drop_item = rospy.ServiceProxy('dropItem', dropItem)
+    check_item_attached = rospy.ServiceProxy('checkItemAttached', checkItemAttached)
+
+    try:
+        
+        response = start_pump(300, 1000)
```

Comment:
Will be good here, instead of passing the raw numbers, to assign these numbers to named variables and pass these variables to the service instead. This will help the reader understand what 300 and 1000 is.